### ISSUE TITLE
Concerning definition for running_loss

### ISSUE THREAD

**Issue by phucdoitoan** [Andras7/word2vec-pytorch#8]
It is not an issue. I just want to ask why do you use running_loss = running_loss*0.9 + loss.item()*0.1 for monitoring the loss during training?
Do you have any special reason for this?
Isnt it conventional to monitor the average loss after each epoch (in this case, after each iteration)?

**Comment by Andras7** [comment]
Hi,
I've used the moving average because it seemed more useful for me (it forgets the past), but feel free to use anything else.
